ntp-proto/src/filter.rs

```diff
@@ -402,26 +437,37 @@ fn find_interval(chime_list: &[CandidateTuple]) -> (NtpDuration, NtpDuration) {
 
         //  If the number of midpoints is greater than the number
         //  of allowed falsetickers, the intersection contains at
-        //  least one truechimer with no midpoint.  If so,
-        //  increment the number of allowed falsetickers and go
-        //  around again.  If not and the intersection is
-        //  non-empty, declare success.
+        //  least one truechimer with no midpoint.  If so, increment
+        //  the number of allowed falsetickers and go around again.
         if found > allow {
```

Comment:
According to the spec on page 43 point 5 we should reject even if found < allow (only accept if found == allow). Is this a conscious deviation from that part?

Reply:
yes, the logic in practice is such that we could jump over a `!=`, so we use `>` to be safe